src/components/mixins/DownloadMixin.vue

```diff
@@ -41,9 +42,11 @@ export default class DownloadMixin extends Vue {
 
     async downloadCompletedCallback(downloadedMods: ThunderstoreCombo[], downloadId: number): Promise<void> {
         try {
+            this.$store.commit('download/updateDownload', {downloadId, status: DownloadStatusEnum.INSTALLING});
```

Comment:
Thoughts on having custom mutators for these, e.g. `this.$store.commit('download/setInstalling', downloadId);`, and others for done and failed? Would be a bit shorter and wouldn't have to import the enum in all UI components.

Reply:
Yess, I'll do that